fastlane_bot/helpers/txhelpers.py

```diff
@@ -798,36 +798,16 @@ def approve_token_for_arb_contract(self, token_address: str, approval_amount: in
             else:
                 return None
 
-    def _get_layer_one_gas_fee_loop(self, rawTransaction) -> Decimal:
+    def _get_layer_one_gas_fee(self, rawTransaction) -> Decimal:
         """
-        Returns the expected layer one gas fee for a layer 2 Optimism transaction
+        Returns the expected layer one gas fee for a Mantle (layer two) transaction
```

Comment:
Can you change this to Mantle, Base, and Optimism?

Reply:
Semantic: `rawTransaction` --> `raw_transaction` (also in the documentation below)